src/dispatchers.ts

```diff
@@ -463,6 +463,53 @@ export async function handleAnnoucedCreate(
     await addToList(ctx.data.db, ['inbox'], create.id.href);
 }
 
+export const createHandleUndo = (accountService: AccountService) =>
+    async function handleUndo(ctx: Context<ContextData>, undo: Undo) {
+        ctx.data.logger.info('Handling Undo');
+
+        if (!undo.id) {
+            ctx.data.logger.info('Undo missing an id - exiting');
+            return;
+        }
+
+        const follow = await undo.getObject();
+
+        const isFollow = follow instanceof Follow;
+
+        if (!isFollow) {
+            ctx.data.logger.info('Undo does not contain a Follow - exiting');
+            return;
+        }
+
+        if (!follow.actorId || !follow.objectId) {
+            ctx.data.logger.info('Undo contains invalid Follow - exiting');
+            return;
+        }
+
+        const unfollower = await accountService.getAccountByApId(
+            follow.actorId.href,
+        );
+        if (!unfollower) {
+            ctx.data.logger.info('Could not find unfollower');
+            return;
+        }
+        const unfollowing = await accountService.getAccountByApId(
+            follow.objectId.href,
+        );
+        if (!unfollowing) {
+            ctx.data.logger.info('Could not find unfollowing');
+            return;
+        }
+
+        await ctx.data.globaldb.set([undo.id.href], await undo.toJsonLd());
+
+        await removeFromList(ctx.data.db, ['following'], follow.objectId.href);
```

Comment:
We shouldn't need to do this right? All the followers / following logic should be using the `follows` table now (i think 😅)